fe/fe-core/src/main/java/org/apache/doris/httpv2/rest/TableSchemaAction.java

```diff
@@ -67,41 +77,74 @@ protected Object schema(
             String fullDbName = getFullDbName(dbName);
             // check privilege for select, otherwise return 401 HTTP status
             checkTblAuth(ConnectContext.get().getCurrentUserIdentity(), fullDbName, tblName, PrivPredicate.SELECT);
-            OlapTable table;
+            CatalogIf catalog;
+            DatabaseIf db;
+            TableIf table;
             try {
-                Database db = Env.getCurrentInternalCatalog().getDbOrMetaException(fullDbName);
-                table = (OlapTable) db.getTableOrMetaException(tblName, Table.TableType.OLAP);
-            } catch (MetaNotFoundException e) {
+                catalog = Env.getCurrentEnv().getCatalogMgr().getCatalog(InternalCatalog.INTERNAL_CATALOG_NAME);
+                db = catalog.getDbOrAnalysisException(fullDbName);
+                table = db.getTableOrAnalysisException(tblName);
```

Comment:
Can we use `db.getTableOrMetaException(tblName)`?

Reply:
@stalary Thanks for your comment. This patch is to support querying doris `view` through spark, but there are some problems. I will close this PR.